src/include/duckdb/main/query_profiler.hpp

```diff
@@ -142,6 +144,10 @@ struct QueryMetrics {
 	Profiler attach_replay_wal_latency;
 	//! The timer for running checkpoints.
 	Profiler checkpoint_latency;
+	//! The timer for the WAL writes
+	Profiler commit_write_wal_latency;
+	//! The total times WAL is replayed
```

Comment:
`The total number of entries to replay in the WAL.` (just for clarification, it's a bit confusing in the loop, but basically every time we're reading an entry and replaying it, we then increment the offset in the file, so that on the next iteration its the next entry)